src/models/utci.js

```diff
@@ -0,0 +1,619 @@
+import { units_converter, valid_range, round } from "../utilities/utilities.js";
+
+const stress_categories = {
```

Comment:
Better to use an array and do a binary search over it. @YuchenMu had a similar implementation for this. you can talk with him in how to do this in a better way